src/Symfony/Component/ObjectMapper/ObjectMapper.php

```diff
@@ -25,6 +27,7 @@
  * @experimental
  *
  * @author Antoine Bluchet <soyuka@gmail.com>
+ * @author Devoton <oton.traore@gmail.com>
```

Comment:
I don't think we use multiple `author` tags do we?

Reply:
we can yes